src/ambianic/pipeline/ai/fall_detect.py

```diff
@@ -222,6 +230,51 @@ def get_line_angles_with_yaxis(self, pose_dix):
         
         return (l_angle, r_angle)
 
+    def estimate_spinalVector_score(self, pose):
```

Comment:
@bhavikapanara awesome implementation of the unified spinalVectorScore algorithm!!

Reply:
thanks @ivelin